extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaBuildTimeConfig.java

```diff
@@ -23,9 +24,22 @@ public class KafkaBuildTimeConfig {
     @ConfigItem(name = "snappy.enabled", defaultValue = "false")
     public boolean snappyEnabled;
 
+    /**
+     * Whether or not the Kafka is running in RHOSAK.
+     */
+    @ConfigItem(name = "rhosak.isUsed", defaultValue = "false")
+    public boolean rhosakIsUsed;
```

Comment:
I don't see this one being used and I would rather drop it (the name doesn't follow our convention). If you need it then we need to discuss the name.

Reply:
fixed